Resources/Prototypes/Entities/Mobs/Cyborgs/base_borg_chassis.yml

```
@@ -45,7 +45,7 @@
   - type: MobThresholds
     thresholds:
       0: Alive
-      100: Critical
+      110: Critical
```

Revert to 100 for direction.


```suggestion
      100: Critical
```